src/AllergyIntolerance-3-0/Cert/_reference/AllergyIntolerance20/nl-core-practitionerrole-cio-1111111210100.xml

```diff
@@ -35,10 +35,6 @@
          </table>
       </div>
    </text>
-   <identifier>
```

Comment:
Deze handmatig toegevoegde identifiers vallen er nu weer uit. Is dat de bedoeling?

Reply:
Ai, dat had ik gemist. Never mind.